modules/gsp-cluster/variables.tf

```diff
@@ -144,3 +144,28 @@ variable "sealed_secrets_private_key_pem" {
   description = "Sealed secrets private key"
   type        = "string"
 }
+
+variable "github_teams" {
```

Comment:
instead of using github teams here ... maybe we could use a list of github users, since we already have the list of github user names in the `users/*.yaml` and could automate generation of the list for each team from the same master list that determines who is authorized to access the namespaces?

But I don't want to block this PR ... seems like we could come back and change this pretty easily if we wanted to later